task3.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+#include <string.h>
+#include <locale.h>
+
+int main()
+{
+    FILE *file = fopen("input.txt", "r");
+    if (file == NULL)
+    {
+        printf("File not found!");
+        return 1;
+    }
+
+    int frequence[255] = { 0 };
+    while(!feof(file))
+    {
+        ++frequence[fgetc(file)];
+    }
+    fclose(file);
+
+    for (int i = 0; i < 255; ++i)
+    {
+        if (frequence[i])
+        {
+            if ((char) i == '\n')
+            {
+                printf("Symbol (backslash)n: %d times\n", i, frequence[i]);
```

Comment:
```suggestion
                printf("Symbol \\n: %d times\n", frequence[i]);
```

1.  Специальные символы можно указать в строке, если экранировать их символом `\`. Например, кавычка - `\"`, обратный слеш - `\\`.
2. Лишний аргумент передан в `printf`.